<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageOuvertureWidget</class>
 <widget class="QWidget" name="MessageOuvertureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="MainLayout">
       <item row="3" column="0">
        <widget class="QComboBox" name="message2CBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="message1CBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="message3CBox"/>
       </item>
       <item row="10" column="0">
        <widget class="QComboBox" name="message5CBox"/>
       </item>
       <item row="8" column="0">
        <widget class="QComboBox" name="message4CBox"/>
       </item>
       <item row="12" column="0">
        <widget class="QComboBox" name="message6CBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="time1Spin">
         <property name="suffix">
          <string/>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="time2Spin">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="time3Spin">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="time4Spin">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="time5Spin">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="time6Spin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="time3CBox"/>
       </item>
       <item row="8" column="2">
        <widget class="QComboBox" name="time4CBox"/>
       </item>
       <item row="10" column="2">
        <widget class="QComboBox" name="time5CBox"/>
       </item>
       <item row="12" column="2">
        <widget class="QComboBox" name="time6CBox"/>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="time2CBox"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="time1CBox"/>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QLabel" name="label4">
         <property name="text">
          <string>Quatrième message</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>Troisième message</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Deuxième message</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>Premier message</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="label5">
         <property name="text">
          <string>Cinquième message</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="3">
        <widget class="QLabel" name="label6">
         <property name="text">
          <string>Sixième message</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
